packages/react-heartwood-components/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -36,8 +36,8 @@ const ButtonGroup = (props: Props) => {
 								kind === 'floating'
 									? 'simple'
 									: kind === 'segmented'
-										? 'secondary'
-										: action.kind
+									? 'secondary'
```

Comment:
Unrelated to this prettier fixup but we probably should avoid nested ternaries, they're not very readable; there's a rule for that if we want to use it: https://eslint.org/docs/rules/no-nested-ternary

Reply:
I'm down for that, never said to myself, "I'm sure happy we have nested ternaries". thoughts @kengoldfarb ?